.github/workflows/ds-storybook-deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy Design System Storybook
+
+on:
+  push:
+    paths:
+      - "packages/design-system/**"
+    branches:
+      - "main"
+
+# Change the working directory for all the jobs in this workflow
+defaults:
+  run:
+    working-directory: packages/design-system
+
+jobs:
+  deploy-storybook:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code
+      - uses: actions/checkout@v3
+
+      # Setup node with version 16.x and NPM registry url
+      - uses: actions/setup-node@v1
```

Comment:
Any reason we aren't using `@v3` of this? 🤔

Reply:
That's the latest version

Reply:
Not sure I understand. We want Node 16, so if we change this to `v3`, and change `node-version: "16.x"` to `node-version: 16`, it should work just as well? But with the current version of the setup-node action?

Reply:
Oh got it sorry, I got your question wrong. I will update this